src/utils.jl

```diff
@@ -30,12 +30,12 @@ Base.convert(::Type{<:Vector}, v::ZeroVector{T}) where {T} = Vector(v)
 
 Base.convert(::Type{ZeroVector{T}}, v::ZeroVector) where {T} = ZeroVector{T}(length(v))
 
-for T = (:AbstractArray, :Number), op = (:+, :-)
-    @eval begin
-        Base.@deprecate ($op)(x::$T, v::ZeroVector) broadcast($op, x, v)
-        Base.@deprecate ($op)(v::ZeroVector, x::$T) broadcast($op, v, x)
-    end
-end
+# for T = (:AbstractArray, :Number), op = (:+, :-)
```

Comment:
Revert or remove?

Reply:
ah indeed thanks, I was so happy to push once all tests passed that I forgot to remove it definitely